notebook/static/notebook/js/keyboardmanager.js

```diff
@@ -42,6 +42,46 @@ define([
         this.edit_shortcuts = new keyboard.ShortcutManager(undefined, options.events, this.actions, this.env);
         this.edit_shortcuts.add_shortcuts(this.get_default_common_shortcuts());
         this.edit_shortcuts.add_shortcuts(this.get_default_edit_shortcuts());
+
+
+        this.config = options.config;
+        var that = this;
+
+        this.config.loaded.then(function(){ 
+            var edit_unbind;
+
+            try {
+                edit_unbind = that.config.data.keys.edit.unbind;
+            } catch (e) {
+                if (e instanceof TypeError) {
+                    edit_unbind = [];
+                } else {
+                    throw e;
+                }
+            }
+
+            edit_unbind.forEach(function(u){that.edit_shortcuts.remove_shortcut(u);});
```

Comment:
Ah, you're right. I think it's because a function has the scope of the object it was defined on, so by default `remove_shortcut` has the scope of the `edit_shortcuts` object.

ES6 gives a very concise way to get around most `this` problems with arrow functions, but alas, it's [not supported by most browsers yet](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Browser_compatibility).